config/formats.ts

```diff
@@ -2933,3 +3006,9 @@ export const Formats: (FormatsData | {section: string, column?: number})[] = [
 		ruleset: ['HP Percentage Mod', 'Cancel Mod'],
 	},
 ];
+
+<<<<<<< HEAD
+export const Formats: FormatList[] = merge(MainFormats, SideFormats);
+=======
+export const Formats: (FormatsData | {section: string, column?: number})[] = merge(MainFormats, SideFormats);
+>>>>>>> d5b5091b1a92ea492c894dacd49819887a77e15a
```

Comment:
i think u missed a merge conflict

Reply:
Also, would it be beneficial to have the side formats be in a separate file?

Reply:
My bad, thank you.

Also yes. Very yes. As someone who runs a side server, I get all my merge conflicts in formats. Github doesn't like it.

Reply:
Oh, im blind. this was already in a separate file, i was suggesting that it was moved to one. oops